.buildkite/release-automation/pre_release.rayci.yml

```diff
@@ -0,0 +1,42 @@
+group: Pre-release checks
+steps:
+  - label: "Check release blockers"
+    key: check-release-blockers
+    depends_on: forge
+    job_env: forge
+    instance_type: small_branch
+    commands:
+      - bazel run //ci/ray_ci/automation:weekly_green_metric -- --check
+
+  - block: "Trigger tests & image builds"
+    depends_on:
+      - step: check-release-blockers
+        allow_failure: true
+
+  - label: "Check commit hash"
+    key: check-commit-hash
+    job_env: forge
+    commands:
+      - if [[ "${BUILDKITE_COMMIT}" != "$(git rev-parse HEAD)" ]]; then echo "Commit hash mismatch"; exit 1; else echo "Commit hash matches"; fi
```

Comment:
could you move this into a bash script on the repo?

Reply:
Triggered a build and checking commit hash works normally: https://buildkite.com/ray-project/release-automation/builds/184#018df6bb-4fe1-44f0-a9e0-a925dbe05c9f